components/org.wso2.carbon.identity.oauth.uma.resource.service/src/test/java/org/wso2/carbon/identity/oauth/uma/resource/service/dao/util/DAOUtils.java

```diff
@@ -27,12 +27,15 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.spy;
+
 public class DAOUtils {
```

Comment:
Add class comment and method comments.

Reply:
fixed: https://github.com/wso2-extensions/identity-oauth-uma/pull/68